client/src/i18n/en/report.json

```diff
@@ -50,6 +50,10 @@
     "CONFIGURATION": "Report Configuration",
     "DELETE": "Delete Report",
     "DOWNLOAD": "Download",
+    "EMPLOYEE_STANDING" : {
+      "DESCRIPTION": "",
```

Comment:
Your french translation is:
> Ce module permet de connaître la situation financière d'un employé"

So this could be "This module shows the financial situation of an employee".